### DISCUSSION TITLE
Should I code this?

### DISCUSSION THREAD

**Discussion by samuellouf** [TurboWarp/extensions#997] [Ideas]
You can fill [this Google Form](https://forms.gle/ngQKro6RP4kceA3x6) to tell me if I should make this extension.

**Comment by yuri-kiss** [comment]
2 people have made this, but you can have a go, it could be a fun learning experience!

**Reply by samuellouf** [reply]
ok

**Comment by DNin01** [comment]
There's also a [Polls category](https://github.com/TurboWarp/extensions/discussions/categories/polls), but people can also just upvote and discuss.
